plugins/Open Chat-eb5b1b45-217a-4cd6-b65e-defc1f820f8e.json

```
@@ -0,0 +1,12 @@
+{
+    "ID": "eb5b1b45-217a-4cd6-b65e-defc1f820f8e",
+    "Name": "Open Chat",
+    "Description": "Open AI chat apps with prefilled prompts. Type 'chat' followed by your message to start conversations instantly.",
+    "Author": "Jesper Svensson",
+    "Version": "1.1.0",
```

the latest release says 1.1.1
```suggestion
    "Version": "1.1.1",
```